mypy/dmypy_server.py

```diff
@@ -95,6 +95,9 @@ def __init__(self, flags: List[str]) -> None:
             sys.exit("dmypy: start/restart does not accept sources")
         if options.report_dirs:
             sys.exit("dmypy: start/restart cannot generate reports")
+        if options.junit_xml:
+            sys.exit("dmypy: start/restart does not support --junit-xml; "
```

Comment:
Alternatively, we could keep ignoring this flag silently, or print a warning without exiting. Both would be slightly more convenient for the Dropbox use case, but sound like a possible cause for confusion for other users.

Reply:
Probably cleanest to fail on it